packages/dialtone/lib/build/less/components/radio-checkbox.less

```diff
@@ -148,6 +148,16 @@
     flex-direction: column;
 }
 
+.d-radio__copy-description {
+    display: flex;
+    box-sizing: border-box;
+    color: var(--dt-color-foreground-tertiary);
+    font-size: var(--dt-font-size-100);
+    font-family: inherit;
+    line-height: var(--dt-font-line-height-400);
+    fill: currentColor;
+}
```

Comment:
`d-description`

Reply:
Ok, there is no changes on `radio-checkbox.less`

Reply:
No, since `d-description` is not a utility I removed the change.